src/common/binary.cpp

```diff
@@ -109,7 +109,7 @@ void loadItems(const std::string& fileName, std::vector<io::Item>& items) {
 
 io::Item getItem(const void* current, const std::string& varName) {
   std::vector<io::Item> items;
-  loadItems(current, items);
+  loadItems(current, items, /*mapped=*/true);
```

Comment:
How sure are we this is always OK?

Reply:
I have tested this with the iOS build and with through our quicksand paths and have had no issues. If there are other scenarios that you know of that you want to test to ensure they don't break we can try and test them as well

Reply:
in binary.cpp in the loadItems function there is a comment that indicates that perhaps there may be some hardware-specific compatibility we should test?
// @TOOD: verify this actually works for the hardware-specific ones like intgemm8avx2

Reply:
if you are uncomfortable with this, we can also pass the mapped variable through the callchain to let callers set it as they see fit. we need this change in iOS otherwise loading/parsing the YAML is too slow

Reply:
There have been a lot of changes in the mmapping interface in internal master, so we will take a look in the merge PR what is going on there.